src/Refinery/Integer/Group.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * @author  Niels Theen <ntheen@databay.de>
+ */
+namespace ILIAS\Refinery\Integer;
+
+use ILIAS\Data\Factory;
+use ILIAS\Refinery\Integer\Constraints\GreaterThan;
+use ILIAS\Refinery\Integer\Constraints\LessThan;
+
+class Group
+{
+	/**
+	 * @var Factory
+	 */
+	private $dataFactory;
+
+	/**
+	 * @var \ilLanguage
+	 */
+	private $language;
+
+	public function __construct(Factory $dataFactory, \ilLanguage $language)
+	{
+		$this->dataFactory = $dataFactory;
+		$this->language = $language;
+	}
+
+	public function isGreaterThan(int $minimum) : GreaterThan
```

Comment:
Please add a short description here. Of course the method name is verbose in this case, but there might be future transformations which need more explanations in the docs.